.github/workflows/e2e_tests.yaml

```diff
@@ -141,7 +141,7 @@ jobs:
           run-bootstrap: false
           melos-version: '3.0.1'
       - name: 'Bootstrap package'
-        run: melos bootstrap --scope tests
+        run: melos bootstrap --scope tests && melos bootstrap --scope "cloud_firestore*"
```

Comment:
any reason why this now fails on Firestore? I noticed this on my own PR 🤔

Reply:
I've refactored the CI to use the same thing on test and cloud_firestore, but forgot to bootstrap those with melos. It would fail because it uses the pub.dev version :/